src/test-page.js

```diff
@@ -77,10 +77,15 @@ const Badge = ({ doi, layout, showLabels, forceCollapse, placement, small, autol
     data-campaign='test'
     data-force-collapse={forceCollapse}
     data-rewardful-id={rewardfulID}
+    data-show-tally={showTally}
+    data-show-section-tally={showSectionTally}
+    data-chart-type={chartType}
+    data-show-logo-section-tally={showLogoSectionTally}
+    data-layout-section-tally={layoutSectionTally}
```

Comment:
I will comment on these here since it's easier to see the API as the user sees them.

Some suggestions for clarity and reduction are:

**one**
Can we simply have one show logo param and controls whether the logo is show at all?
I feel like it doesn't have to be specific to the section tally and maybe is a bit confusing to a user who see's it as a option for one and not another.

**two**
I still am not sure about this but maybe the API naming scheme should be `{type_of_tally}-{param}`?
for instance :

```suggestion
    data-tally-show={showTally}
    data-section-tally-show={showSectionTally}
    data-section-tally-layout={layoutSectionTally}
```

I really don't know if it makes things clearer at all... but then maybe for the user they have a bit more of an alphabetical scheme in their head for the config?

Reply:
1. Actually yes, just thought the same, I think I did it separately since I was treating it separately but it can be composed to just 1.
2. Sure, wfm

Reply:
Updated!